cu-calendar/cu-calendar-test.tex

```diff
@@ -0,0 +1,28 @@
+\documentclass[12pt]{article}
+\usepackage{cu-calendar}
+\begin{document}
+
+\makeatletter
+\def\assertSame#1#2{%
+    \edef\@aa{#1}%
+    \edef\@bb{#2}%
+    \ifx\@aa\@bb\else
+        \errmessage{TEST FAILED: #1: "\@aa" expected "\@bb"}%
+    \fi
+}%
+
+\assertSame{\cu@jdnFromJulianDate{2016}{3}{30}}{2457491}
+\assertSame{\cu@jdnFromGregorianDate{2016}{3}{30}}{2457478}
+
+\assertSame{\cu@calendarJulian{2457491}}{2016-3-30}
+\assertSame{\cu@calendarGregorian{2457478}}{2016-3-30}
+
+\assertSame{\cu@julianFromGregorian{2016}{3}{30}}{2016-3-17}
+\assertSame{\cu@gregorianFromJulian{2016}{3}{17}}{2016-3-30}
+
+\assertSame{\cuJulianDate{2016}{3}{30}}{17 ма́рта л. 2016}
+\assertSame{\cuGregorianDate{2016}{3}{30}}{30 ма́рта л. 2016}
+
```

Comment:
I think for a full test we need to test the complicated cases, for example, make sure that 5 October 1582 Julian == 15 October 1582 Gregorian. Also, what happens when a user specifies a date like 14 October 1582 Gregorian? Does this exist?
Also, check the complicated leap years, for example, make sure that 2100 *is* a Julian leap year (29 February 2100 exists) but not a Gregorian leap year (29 February 2100 does not exist).